app/src/seed_script.py

```diff
@@ -0,0 +1,195 @@
+                                sed risus et gravida. Curabitur sodales id dolor vel interdum.
+                            </p>
+
+                            <p>
+                                Vestibulum scelerisque mattis consequat. In eget urna tellus. Morbi velit nunc, hendrerit vel lorem 
+                                quis, tincidunt gravida neque. Pellentesque egestas enim nec lacus tempor, sed bibendum felis lobortis. 
+                                Aenean vestibulum vel leo in semper. Integer gravida luctus ornare. Nullam magna tellus, sagittis sed 
+                                dapibus sed, congue quis turpis. Quisque id enim in arcu faucibus scelerisque. Suspendisse odio massa, 
+                                imperdiet et urna a, aliquam aliquet neque. Phasellus suscipit ex quis libero finibus, porta efficitur 
+                                augue gravida. Etiam ac pretium urna. Pellentesque posuere hendrerit turpis sed suscipit. Suspendisse 
+                                ultrices, augue in tristique ultricies, ligula diam facilisis justo, eget consequat mauris ipsum sit 
+                                amet nunc. Etiam non nisl justo. Donec dignissim ullamcorper maximus. Fusce rhoncus volutpat augue.  
+                            </p>
+                            """,
+            textbook_name="sample.pdf",
+            textbook_pages="<li>Chapter 3 - Read Pages 44 to 66</li><li>Complete Exercises 1-10</li>",
+            practice_content="Spaced repetition learning here, maybe through some mini interactable question module here (for no marks, just practice)?",
+            topic_id=module1_topic2.id
+        )
+        db.session.add(module1_topic2_lesson)
+        db.session.commit()
+
+        module1_topic3 = Topic(name="Prime Factorization", module_id=module1.id)
+        db.session.add(module1_topic3)
+        db.session.commit()
+
+        module1_topic3_lesson = Lesson(
+            title="Factors & Multiples",
+            learning_objective="<li>Understand and Identify Factors</li><li>Apply factorization in real world contexts</li><li>Utilize factors to Simplify Fractions</li><li>Explore and apply prime factorization</li>",
+            video_filename="video.mp4",
+            thumbnail_filename="thumbnail.png",
+            textbook_name="sample.pdf",
+            textbook_pages="<li>Chapter 3 - Read Pages 44 to 66</li><li>Complete Exercises 1-10</li>",
+            practice_content="Spaced repetition learning here, maybe through some mini interactable question module here (for no marks, just practice)?",
+            topic_id=module1_topic3.id
+        )
+        db.session.add(module1_topic3_lesson)
+        db.session.commit()
+
+        create_file_paths(course.id, module1.id, module1_topic1.id)
+
+        module2 = Module(name="Patterns", course_id=course.id)
+        db.session.add(module2)
+        db.session.commit()
+
```

Comment:
Just to have another module with something in it, I'd add a corresponding topic and lesson:
```python
module2_topic1 = Topic(name="Pattern Rules", module_id=module2.id)
db.session.add(module2_topic1)
db.session.commit()

module2_topic1_lesson = Lesson(
    title="Pattern Rules",
    learning_objective="<li>Understand and Identify Factors</li><li>Apply factorization in real world contexts</li><li>Utilize factors to Simplify Fractions</li><li>Explore and apply prime factorization</li>",
    video_filename="video.mp4",
    thumbnail_filename="thumbnail.png",
    textbook_name="sample.pdf",
    textbook_pages="<li>Chapter 3 - Read Pages 44 to 66</li><li>Complete Exercises 1-10</li>",
    practice_content="Spaced repetition learning here, maybe through some mini interactable question module here (for no marks, just practice)?",
    topic_id=module2_topic1.id
)
db.session.add(module2_topic1_lesson)
db.session.commit()

create_file_paths(course.id, module2.id, module2_topic1.id)
```